acceptance-tests/coreclr.mk

```diff
@@ -4003,6 +4001,14 @@ CORECLR_DISABLED_TEST_CS_SRC += 	\
 # this test makes no sense, the expected pattern is the English one for all cultures
 CORECLR_DISABLED_TEST_CS_SRC += $(CORECLR_PATH)/tests/src/Regressions/coreclr/0584/test584.cs
 
+# Requires precise stack scanning
+CORECLR_DISABLED_TEST_CS_SRC +=        \
+       $(CORECLR_PATH)/tests/src/JIT/Directed/lifetime/lifetime2.cs
+
+# Depends on small array behavior of .net. Mono supports objects > 2Gb on 64bits and the following tests verify for that
+CORECLR_DISABLED_TEST_CS_SRC +=        \
+       $(CORECLR_PATH)/tests/src/JIT/Directed/newarr/newarr.cs
```

Comment:
This is disabled by default AFAIK (you need to pass --enable-big-arrays to ./configure), so it should still fail as .NET?

In any case, I'd prefer ifdef-ing out the "big array" case in the test source instead so that the other cases are still tested.